media-libs/libtiger/libtiger-0.3.4-r1.ebuild

```diff
@@ -27,7 +27,9 @@ src_prepare() {
 
 multilib_src_configure() {
 	ECONF_SOURCE=${S} \
-	econf $(use_enable doc)
+	econf \
```

Comment:
Err, what do you mean? We usually don't indent command after an environment variable because it's the command™.